extensions/ql-vscode/test/unit-tests/model-editor/bqrs.test.ts

```diff
@@ -230,11 +230,11 @@ describe("decodeBqrsToExternalApiUsages", () => {
     ],
   };
 
-  it("extracts api usages", () => {
-    // Even though there are a number of usages with the same number of usages, the order returned should be stable:
+  it("extracts methods", () => {
+    // Even though there are a number of methods with the same number of usages, the order returned should be stable:
     // - Iterating over a map (as done by .values()) is guaranteed to be in insertion order
     // - Sorting the array of usages is guaranteed to be a stable sort
```

Comment:
I couldn't work out if this one should be "methods" too 😅 Ignore me if you already checked this!

Reply:
I honestly don't know for sure, but on second reading I think it's probably meant to be "methods" here too.